src/lib/components/Codelab/_styles.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../../styles/settings/colors';
+
+web-codelab {
+  display: flex;
+  height: 100%;
+}
+
+.web-codelab__headline {
```

Comment:
I know you don't like nesting but it seems sensible to wrap this in at least the CE itself?

Reply:
hm yeah I just noticed the LH gauge does that. I don't think it gives any benefit since BEM already prevents collisions but it might be nice to be consistent with the other custom elements.

I was actually thinking that it might make sense to switch to [CSS Modules](https://github.com/css-modules/css-modules) at some point since they play nice with Sass and could give us essentially the same scoping as Shadow DOM without all the polyfilling.